lib/util/i18n.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+function t (options, key, original, props) {
```

Comment:
hey @vitaliystoliarovcc maybe we should rename options param to i18n? I think it's a bit confusing right now :sweat_smile:

Reply:
Actually options here are object `{t: function(){}, prefix: string}` 

I'll add the documentation 😅

Reply:
done